docs/entities/tokens/non-fungible-token-history-event-0-tx-id.schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "NonFungibleTokenHistoryEventWithTxId",
+  "description": "Non-Fungible Token history event with transaction id",
+  "required": ["event_index", "asset_event_type_id", "tx_id"],
+  "additionalProperties": false,
+  "properties": {
+    "sender": {
+      "type": "string"
+    },
+    "recipient": {
+      "type": "string"
+    },
+    "event_index": {
+      "type": "integer"
+    },
+    "asset_event_type_id": {
```

Comment:
Do we return `asset_event_type_id` and/or its number value elsewhere? I think we generally treat the integer as an internal datastore implementation detail, and we run it through `getAssetEventTypeString` to get a useful string/enum to return on responses.

Reply:
Nice catch, a string would be much better here, will do